### PR TITLE
3d sphere

### PR DESCRIPTION
**Pull request by janphilipp-winkler**

### CODE REVIEW

File: components/3DAnimation/Sphere.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <>
+      <mesh ref={mesh}>
+        <sphereGeometry args={[1.4, 512, 512]} />
+        <shaderMaterial
+          fragmentShader={fragmentShader}
+          vertexShader={vertexShader}
+          uniforms={uniforms}
+        />
+      </mesh>
+    </>
```
Comment (MerleMak) [inline components/3DAnimation/Sphere.js:98]: the Fragments `<> </>` are not needed here

**Review by MerleMak — COMMENTED** [review]
I looks good I guess 😅
I don't understand much but as long as it works, all great. 
It looks very impressive. 
All the checks failed tho, did you got that fixed now?

File: components/3DAnimation/Sphere.js (hunk 1 above)
Comment (SevketErsan) [reply components/3DAnimation/Sphere.js:98]: The problem is inconsistency in shader file names. 'vertexshader' versus 'vertexShader'. While there were no issues locally, Vercel raised a problem.

Comment (SevketErsan) [reply components/3DAnimation/Sphere.js:98]: Yes, the fragments are unnecessary. We had used them to add the Glow Effect component, but the effect was a bit excessive. So now they are unnecessary.

**Pull request merged**